cmd/root/bundle_test.go

```diff
@@ -83,7 +83,7 @@ func TestBundleConfigureWithNonExistentProfileFlag(t *testing.T) {
 	cmd.Flag("profile").Value.Set("NOEXIST")
 
 	b := setup(t, cmd, "https://x.com")
-	assert.PanicsWithError(t, "no matching config profiles found", func() {
+	assert.Panics(t, func() {
```

Comment:
Are we failing with different error now?

Reply:
Yes, this used to be triggered by the validator, now it is triggered by the profile loader in the SDK.

Before it was `errNoMatchingProfiles` from `libs/databrickscfg`.

Now it is `<path to>/.databrickscfg has no NOEXIST profile configured`.